### DISCUSSION TITLE
幻想卡牌建议收集！

### DISCUSSION THREAD

**Discussion by HibiKier** [HibiKier/zhenxun_bot#1804] [Polls]
[来自幻想](https://github.com/HibiKier/fantasy_card/blob/main/README.md)

**Comment by Gzcay** [comment]
什么时候能用上呀 看着好好玩

**Comment by HibiKier** [comment]
集卡容易，对战大工程

**Comment by xihuantudi** [comment]
你是想做成游戏大厅那样子？还要搞段位和匹配机制？

**Comment by HibiKier** [comment]
> 你是想做成游戏大厅那样子？还要搞段位和匹配机制？

画大饼画大饼

**Comment by Judp28** [comment]
期待，插眼

**Comment by HibiKier** [comment]
![95763057](https://user-images.githubusercontent.com/45528451/177930070-3b768294-49b9-4054-8d52-ea449ce7deed.png)
经过压缩后的牌变糊了....虽然还能看

**Comment by xihuantudi** [comment]
> ![95763057](https://user-images.githubusercontent.com/45528451/177930070-3b768294-49b9-4054-8d52-ea449ce7deed.png) 经过压缩后的牌变糊了....虽然还能看

我只能说，太TM酷了

**Comment by shinianj** [comment]
我觉得，基于目前go无法使用临时对话的情况，玩家之间的对决可能不是很好写。在我看来，卡牌游戏如果作为对决，那么重要的是随机性、临时应对的能力以及计算斩杀等游戏性，那么明牌展示我觉得已经是掐断了玩家对决这条路了，那么可能的办法就是设计bot作为头目进行的人机对战，在此基础上，我想的是设计难度档位以及玩家合作达成互帮互助打倒bot，因为bot的牌是可以无需显示的（甚至可以下点小手段），不过至于玩家上限和bot的难度设计，那我不大好说。

其次的一点，我希望卡牌基于稀有度进行属性分类，可以进行上下位的适当替换，高稀有度的卡牌应当是能提供以下功能中的一项功能的：
1.高灵活性，能够解决多数低稀有度卡造成的场面或是一般情况下的困境 2.高决定性，能够成为卡组核心，围绕他进行一系列的斩杀操作 3.高功能性，能够在场面上给予对方一定的棘手buff，并能有一定的持久性。
（提一手高稀有度卡牌要避免的是，同时拥有以上两种的功能，否则过于人权卡会导致谁都玩的不爽）
 
余下的是本人的游戏设想：游戏分为进攻模式与防守模式。
进攻模式会将bot的牌库进行循环，而每个玩家只有自己所带的一套牌在牌库中，若玩家血量清零或牌库与手牌消耗完，以及场面没有任何会对对战产生影响的卡牌，将会确认该玩家退场，所有玩家退场将判失败；若bot血量清零，则为全玩家胜利。
防守模式则相反，但bot的牌组会有更强的攻击性和威胁性。
将玩家和bot设为主，而每人的场面上只能占有一位角色，玩家和bot可以在后方对盟友、角色、玩家本身进行加护、治疗、附魔等操作（具体还得看阿咪你了），当场上角色退场时，需要一回合或更久的后备角色上场时间（我觉得可以设置随行这种直接上场替补的词条了）。我看卡牌还有右上角的属性设计，这个点应该可以造成更大的场面混乱（嗯，希望不会太乱）
以上个人想法，仅供参考（很抱歉，阿咪，我经不起骂，请别骂我）

**Comment by HibiKier** [comment]
> 我觉得，基于目前go无法使用临时对话的情况，玩家之间的对决可能不是很好写。在我看来，卡牌游戏如果作为对决，那么重要的是随机性、临时应对的能力以及计算斩杀等游戏性，那么明牌展示我觉得已经是掐断了玩家对决这条路了，那么可能的办法就是设计bot作为头目进行的人机对战，在此基础上，我想的是设计难度档位以及玩家合作达成互帮互助打倒bot，因为bot的牌是可以无需显示的（甚至可以下点小手段），不过至于玩家上限和bot的难度设计，那我不大好说。
> 
> 其次的一点，我希望卡牌基于稀有度进行属性分类，可以进行上下位的适当替换，高稀有度的卡牌应当是能提供以下功能中的一项功能的： 1.高灵活性，能够解决多数低稀有度卡造成的场面或是一般情况下的困境 2.高决定性，能够成为卡组核心，围绕他进行一系列的斩杀操作 3.高功能性，能够在场面上给予对方一定的棘手buff，并能有一定的持久性。 （提一手高稀有度卡牌要避免的是，同时拥有以上两种的功能，否则过于人权卡会导致谁都玩的不爽）
> 
> 余下的是本人的游戏设想：游戏分为进攻模式与防守模式。 进攻模式会将bot的牌库进行循环，而每个玩家只有自己所带的一套牌在牌库中，若玩家血量清零或牌库与手牌消耗完，以及场面没有任何会对对战产生影响的卡牌，将会确认该玩家退场，所有玩家退场将判失败；若bot血量清零，则为全玩家胜利。 防守模式则相反，但bot的牌组会有更强的攻击性和威胁性。 将玩家和bot设为主，而每人的场面上只能占有一位角色，玩家和bot可以在后方对盟友、角色、玩家本身进行加护、治疗、附魔等操作（具体还得看阿咪你了），当场上角色退场时，需要一回合或更久的后备角色上场时间（我觉得可以设置随行这种直接上场替补的词条了）。我看卡牌还有右上角的属性设计，这个点应该可以造成更大的场面混乱（嗯，希望不会太乱） 以上个人想法，仅供参考（很抱歉，阿咪，我经不起骂，请别骂我）

很好的想法，在群里玩家对战基本为自动，私聊为手动模式，组团攻略世界boss(真寻)这个想法不错捏

**Comment by Judp28** [comment]
既然决定群内是讨伐头目的战斗模式，我有几点建议。
.关于游戏战斗机制。可以是小队进副本制，开团挑战副本的为队长，接下来进团的为队员。队长提供自己卡包内的角色卡站场，小队成员提供自己卡包内的特殊效果卡为角色卡上buff与ai战斗。既然是合作挑战，可设置每次进本需要消耗金币/每日挑战限制/战败惩罚。如不加以限制，重复刷本容易腻。
.游戏难度梯度。类似于探索地下城，可以设置三个普通关卡一个boss关卡循环，一定要有梯度，也要有奖励机制，如普通关卡掉落金币，boss难度概率掉落boss卡包等。
(战斗过程如果太长，设置为合并转发消息，群聊内刷屏影响观感)
仅代表个人观点
